src/corebluetooth/internal.rs

```diff
@@ -229,6 +232,15 @@ impl CBPeripheral {
                 .set_reply(CoreBluetoothReply::Connected(services));
         }
     }
+
+    pub fn confirm_disconnect(&mut self) {
+        self.disconnected_future_state
+            .take()
+            .unwrap()
+            .lock()
+            .unwrap()
+            .set_reply(CoreBluetoothReply::Disconnected);
```

Comment:
Could you just use `CoreBluetoothReply::Ok` here, rather than introducing a new variant?